yeelight/main.py

```diff
@@ -102,6 +124,24 @@ def last_properties(self):
         """
         return self._last_properties
 
+    @property
+    def bulb_type(self):
+        """
+        Returns BulbType :py:enum:`BulbType <yeelight.BulbType>` describing type
+        of the bulb. Currently this is Color or White.
+
+        When trying to access before properties are known, the bulb type is unknown.
+
+        :rtype: BulbType
+        :return: The bulb's type.
+        """
+        if not self._last_properties:
+            return BulbType.Unknown
+        if not any([self.last_properties[name] for name in ['ct', 'rgb', 'hue', 'sat']]):
```

Comment:
That's true too. I'll try to change and re-test everything again tomorrow. Currently on something completely else.